app/hotels/src/__tests__/HotelsContext.test.js

```diff
@@ -16,177 +14,78 @@ describe('getAsUtcDate', () => {
     expect(getAsUtcDate('2018-01-01')).toEqual(new Date(Date.UTC(2018, 0, 1)));
   });
 });
-const props = {
-  checkin: DateFormatter(DateUtils().addDays(30)).formatForMachine(),
-  checkout: DateFormatter(DateUtils().addDays(37)).formatForMachine(),
-};
-const getInstance = (additionalProps = {}) =>
-  renderer
-    .create(
-      <>
-        {/*  $FlowExpectedError: Just passing props needed for test to pass */}
-        <HotelsContext.Provider {...props} {...additionalProps}>
-          {null}
-        </HotelsContext.Provider>
-      </>,
-    )
-    .getInstance();
 
-// In the end this is how this date is used, so it makes sense to pass it through this format for asserting results
-const formatForMachine = (date: Date) => DateFormatter(date).formatForMachine();
-
-describe('HotelsContext', () => {
-  describe('constructor', () => {
-    it('validates correctly when there is no error', () => {
-      const wrapper = getInstance();
-      expect(wrapper.state.errors).toEqual({});
-    });
-
-    it('gives error for cityId -1', () => {
-      const wrapper1 = getInstance({ cityId: '-1' });
-      const wrapper2 = getInstance({ cityId: -1 });
-      expect(wrapper1.state.errors).toEqual({ invalidCityId: true });
-      expect(wrapper2.state.errors).toEqual({ invalidCityId: true });
-    });
-
-    it('gives error for checkin date before today', () => {
-      const wrapper = getInstance({
-        checkin: formatForMachine(DateUtils().addDays(-1)),
-        checkout: formatForMachine(DateUtils().addDays(1)),
-      });
-      expect(wrapper.state.errors).toEqual({ beforeToday: true });
-    });
+const getValidateProps = (cityId, checkin, checkout) => ({
```

Comment:
```suggestion
const getValidateProps = ({cityId, checkin, checkout}) => ({
```
Maybe as an object, it would be clearer in the tests below what you're setting when calling `getValidateProps(-1)`?

Reply:
👍